bioimageio/spec/shared/utils.py

```diff
@@ -340,45 +347,14 @@ def _download_uri_to_local_path(uri: typing.Union[nodes.URI, raw_nodes.URI]) ->
     return local_path
 
 
-def resolve_raw_node_to_node(raw_node: GenericRawNode, root_path: os.PathLike, nodes_module: ModuleType) -> GenericNode:
+def resolve_raw_node(raw_node: GenericRawNode, root_path: os.PathLike, nodes_module: ModuleType) -> GenericNode:
```

Comment:
Is this user facing? If it is we should maybe rename to `resolve_raw_resource_description`?

Reply:
this is not user facing and only used in shared.io_ where the node/raw_node names are still used. I'm working on a separate PR to improve the node raw_node distinction and will change the names in io_ there as well. But all of this is not user facing...